sgx_page_cache.c

```diff
@@ -367,6 +367,7 @@ static void sgx_swap_pages(unsigned long nr_to_scan)
 	struct sgx_tgid_ctx *ctx;
 	struct sgx_encl *encl;
 	LIST_HEAD(cluster);
+	struct rw_semaphore *sem;
 
```

Comment:
Most likely OK in this particular case, but I would prefer not assume a type for mmap_lock or mmap_sem, Can u use mmap_read_lock and mmap_read_unlock API instead?

Reply:
Changes applied.  Let me know if this works for you.